2_adventures/001/kotp/04_automated_testing_example/spec/fizz_buzz_spec.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'spec_helper'
+require 'fizz_buzz'
+
+describe FizzBuzz do
+  describe 'Single Numbers' do
+    it 'must return 1 for 1' do
```

Comment:
Yes, the output is similar to the Rspec example above. If you change the `it` call to e.g. `it 'returns "Buzz"` it becomes easier to scan.